.github/workflows/deploy-to-cloud-run.yml

```
@@ -0,0 +1,255 @@
+          --format 'value(status.url)')
+        echo "deployment_url=$URL" >> $GITHUB_OUTPUT
+        echo "Deployment URL: $URL"
+
+    - name: Health check
+      run: |
+        echo "🏥 Performing health check..."
+        sleep 30
+        
+        URL="${{ steps.get-url.outputs.deployment_url }}"
+        echo "Checking health at: $URL/health"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)